.eslintrc.yml

```diff
@@ -23,8 +23,8 @@ settings:
             - static
             - node_modules
           alias:
-            '~js': js
-            '~img': images
+            '#js': js
+            '#img': images
```

Comment:
Using `~` as a custom alias prefix is a bit confusing to begin with, since it has magical meaning (`node_modules`) in many build systems (incl Webpack). A recent change in [css-loader](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#600-2021-07-14) made this more pressing.